subprojects/docs/src/samples/userguide/files/archives/groovy/build.gradle

```diff
@@ -14,6 +14,19 @@ task unpackFiles(type: Copy) {
 }
 // end::unpack-archive-example[]
 
+// tag::unpack-archive-subset-example[]
+task unpackLibsDirectory(type: Copy) {
+    from(zipTree("src/resources/thirdPartyResources.zip")) {
+        include "libs/**"
+        includeEmptyDirs = false
+        eachFile { fcd ->
+            fcd.relativePath = new RelativePath(!fcd.isDirectory(), fcd.relativePath.segments.drop(1))
```

Comment:
You should add an annotation to explain what this line is doing.

Reply:
Also, might be worth mentioning somewhere if this can work with absolute paths or only relative paths.